cmd/jaeger/internal/extension/jaegerquery/server.go

```diff
@@ -81,6 +82,7 @@ func (s *server) Start(ctx context.Context, host component.Host) error {
 	//nolint
 	s.server, err = queryApp.NewServer(
 		s.logger,
+		healthcheck.New(),
```

Comment:
Yes it will - this is consistent with the previous behaviour (where `server.HealthCheckStatus()` was not being consumed), but I can look into reporting this upwards.

I think you are referring to this: https://github.com/open-telemetry/opentelemetry-collector/pull/8169

Reply:
Yes. Feel free to add a TODO for now, it doesn't need to be solved in the scope of this PR